### PR TITLE
Slice interface language support

### PR DESCRIPTION
**Pull request by jolkdarr**
Documentation:
- middleware: https://zeroc.com/
- language: https://doc.zeroc.com/display/Ice37/The+Slice+Language

### CODE REVIEW

File: lib/rouge/lexers/slice.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*- #
+
```
Comment (vidarh) [inline lib/rouge/lexers/slice.rb]: Can you add '# frozen_string_literal: true'?

File: spec/lexers/slice_spec.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*- #
+
```
Comment (vidarh) [inline spec/lexers/slice_spec.rb]: Can you add '# frozen_string_literal: true'?

File: lib/rouge/lexers/slice.rb (hunk 1 above)
Comment (jolkdarr) [reply lib/rouge/lexers/slice.rb]: Done

**Comment by pyrmont** [comment]
@jolkdarr I'm sorry it's taken such a long time to get this. Apologies also for force pushing to your branch. It's been such a long time since you made the original PR that it seemed unfair to ask you to make a couple of minor requests.

Please let me know if anything looks off or if you had any questions. Hopefully we'll be able to get this merged in soon!

**Comment by jolkdarr** [comment]
Thanks @pyrmont for processing the PR.

**Comment by pyrmont** [comment]
@jolkdarr Does it look all right to you? Let me know and if it's OK, I'll merge it in.

**Comment by jolkdarr** [comment]
OK for merging.
Thank you once again.

**Pull request merged**